src/Sinch/Conversation/ConversationRegion.cs

```diff
@@ -5,8 +5,19 @@ namespace Sinch.Conversation
     /// </summary>
     public record ConversationRegion(string Value)
     {
+        /// <summary>
+        ///     US Production
+        /// </summary>
         public static readonly ConversationRegion Us = new("us");
 
+        /// <summary>
+        ///     EU Production
+        /// </summary>
         public static readonly ConversationRegion Eu = new("eu");
+
+        /// <summary>
+        ///     BR Production
+        /// </summary>
+        public static readonly ConversationRegion Br = new("br");
```

Comment:
Can you update (or create) the test related to the URL formatting with the different regions?

Reply:
I've created a ticket for that as it's worth testing each APIs url resolution: https://tickets.sinch.com/browse/DEVEXP-593

Reply:
You said in the other comment that "The change is seemingly small improvement". Why not adding the tests in this PR and create another ticket for it, since you have added a test for the other part of the PR related to the Verification API?

Reply:
Okay, I got your point, I just split the PR

Reply:
@asein-sinch the test for url building will come later with separate PR

Reply:
I don't understand, this is a 1-liner. What is the rationale behind pushing some untested code on the main branch and not writing the test in the same PR?